### DISCUSSION TITLE
[FeatureRequest] Variables: Add Url decode formatting option

### DISCUSSION THREAD

**Discussion by phxnsharp** [grafana/grafana#38373] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Grafana adds a convenient [Percentencode](https://grafana.com/docs/grafana/latest/reference/templating/#percentencode). I would like to do the opposite to make a variable appear in human readable form.

I have a URL in a prometheus label: http://xxxxxxxxxxx.com/data/content/files/workflows/hiragana.pacz?#Model.%E3%82%8F%E3%81%9F%E3%81%97

I have created a variable that reads the available urls and can cut off the #Model.%E3%82%8F%E3%81%9F%E3%81%97 part using a regex. However, I would like to provide the human readable name: Model.わたし in the variable for the user. 

**Why is this needed**:

To provide a good user experience to my dashboard user. Trying to read Model.%E3%82%8F%E3%81%9F%E3%81%97 is relatively challenging for most humans.

**Comment by papagian** [comment]
which datasource do you use?

**Comment by phxnsharp** [comment]
My apologies, I seem to have forgotten environment entirely. I am using grafana/grafana:6.7.2 from Docker Hub with prom/prometheus:v2.17.2 as a data source.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by albertocavalcante** [comment]
Is this feature already available or planned to be implemented? Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
